scripts/init.dspacedb5.sh

```diff
@@ -8,11 +8,11 @@ createuser --username=postgres dspace
 
 echo "Importing clarin-dspace"
 createdb --username=postgres --owner=dspace --encoding=UNICODE clarin-dspace
-psql -U postgres clarin-dspace < ../dump/clarin-dspace-8.8.23.sql &> /dev/null
```

Comment:
you sure we need this?

Reply:
It is very helpful. Without it you must find out that imports were successful with logs that tell you they weren't.

Reply:
this is dspace v5, so no import is being done into this database

Reply:
I mean importing the dump into database